include/libfreenect2/frame_listener.hpp

```diff
@@ -58,12 +58,14 @@ class LIBFREENECT2_API Frame
     BGRX,
     RGBX,
     Gray,
-    Float
+    Float,
+    // Note that if format is 'raw' then 'bytes_per_pixel' actually contains the number of bytes
+    Raw
```

Comment:
It looks like the `Raw` format is not assigned to the frames. Add `xx_frame->format = Frame::Raw` below?